src/main/java/net/bytzo/sessility/mixins/ServerPlayerMixin.java

```diff
@@ -50,7 +54,14 @@ private void postTick(CallbackInfo callbackInfo) {
 			return;
 		}
 
-		var idleTime = Util.getMillis() - this.getLastActionTime();
+    // update lastMotileTime if it is older than a stale lastActionTime
+    var lastActionTime = this.getLastActionTime();
+    if (lastActionTime < actionTimeTracker) {
+      lastMotileTime = actionTimeTracker;
+      actionTimeTracker = lastActionTime;
+    }
```

Comment:
I think we might not need to track the last action time in order to update `lastMotileTime`, as `lastMotileTime` already updates whenever action is taken (`setSessile(false)` is called in `resetLastActionTime`)?

Reply:
I believe you're right, I'll double check that when I get a chance.